R/qenv-eval_code.R

```diff
@@ -9,6 +9,9 @@
 #' @param code (`character`, `language` or `expression`) code to evaluate.
 #' It is possible to preserve original formatting of the `code` by providing a `character` or an
 #' `expression` being a result of `parse(keep.source = TRUE)`.
+#' @param keep_output (`logical(1)`) whether to keep the output of the code evaluation.
```

Comment:
Please expand on the documentation. Currently I don't really know what _keep the output_ actually means. Possibly give an examples and explain the difference between T and F